docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -33,14 +33,51 @@ A new window will open, where you will be able to specify the following:
 
 ![Sys-admin-work-package-new-status](Sys-admin-work-package-new-status.png)
 
+
+
 ## Edit, re-order or remove a work package status
 
-1. To **edit** an existing work package type click on the work package name. 
+1. To **edit** an existing work package status click on the status name. 
 2. To **sort** the status, e.g. in the status drop-down list of the work packages, click on the arrows to move the status up or down in the list.
-3. To **delete** the work package type click on the delete icon.
+3. To **delete** the work package status click on the delete icon.
 
 ![Sys-admin-edit-work-package-status](Sys-admin-edit-work-package-status.png)
 
 <div class="alert alert-info" role="alert">
 **Note**: A new status needs to be activated in the [work package workflows](../work-package-workflows) in order to appear in the status drop down of a work package.
 </div>
+<<<<<<< Updated upstream
```

Comment:
Please remove the merge artifacts

Reply:
Thank you for having a thorough look at this! I removed them